apc/silver/Process HES APC (Comorbidities).py

```diff
@@ -1,38 +1,39 @@
 # Databricks notebook source
 import json
+from functools import reduce
+
 import pandas as pd
 import pyspark.sql.functions as F
 from pyspark.sql import DataFrame
-from functools import reduce
 
 # COMMAND ----------
 
+
 def open_comorbidity_file(file, scores):
-    with open(f"/Volumes/su_data/reference/raw/{file}_comorbidity_scores.json", "r") as f:
-        return (
-            spark.createDataFrame([
+    with open(
+        f"/Volumes/strategyunit/reference/files/{file}_comorbidity_scores.json", "r"
+    ) as f:
+        return spark.createDataFrame(
+            [
                 {
                     "group": v.get("trumps", k),
                     "score_type": s,
                     "score": v["score"][s],
-                    "code": c
+                    "code": c,
```

Comment:
Assume the extra comma is just an auto-styler. 
Is trailing comma okay in this syntax?